server/.env.example

```diff
@@ -1 +1 @@
-SESSION_SECRET=""
+SESSION_SECRET="123e4567-e89b-12d3-a456-426614174000"
```

Comment:
A value does not need to be added here. The goal of this file to to be an example representation of the key-value pairs needed in this application. The expectation is that anyone who wants to use this codebase would then make their own `.env` file, and supply a value to the keys listed in the _example_ file. The reason for this is because we want these ENV key values to be kept secret. We don't share they key value data on github, but just the keys the application is looking for. The README for an application would explain this for others who want to use the codebase for what keys they need to apply for.